src/iframe-helper.js

```diff
@@ -81,16 +81,25 @@ function getListenForSentinel(parentWin, sentinel, opt_create) {
  * @param {!Element} iframe the iframe element who's context will trigger the
  *     event
  * @param {boolean=} opt_is3P set to true if the iframe is 3p.
+ * @param {boolean=} opt_isInsensitive is set to true if data is not considered
+ *     sensitive and event origin check doesn't need to take place
  * @return {?Object<string, !Array<function(!JsonObject, !Window, string)>>}
  */
-function getOrCreateListenForEvents(parentWin, iframe, opt_is3P) {
+function getOrCreateListenForEvents(
+  parentWin,
+  iframe,
+  opt_is3P,
+  opt_isInsensitive) {
   const {origin} = parseUrlDeprecated(iframe.src);
   const sentinel = getSentinel_(iframe, opt_is3P);
   const listenSentinel = getListenForSentinel(parentWin, sentinel, true);
 
   let windowEvents;
   for (let i = 0; i < listenSentinel.length; i++) {
     const we = listenSentinel[i];
+    if (opt_isInsensitive === true) {
+      we.opt_isInsensitive = opt_isInsensitive;
```

Comment:
This is leaking into other `windowEvents` holders. This should only be happening at creation time (below), not here.